examples/TestDriver/ios/Podfile.lock

```diff
@@ -105,11 +105,11 @@ SPEC CHECKSUMS:
   DoubleConversion: bb338842f62ab1d708ceb63ec3d999f0f3d98ecd
   Folly: c89ac2d5c6ab169cd7397ef27485c44f35f742c7
   glog: e8acf0ebbf99759d3ff18c86c292a5898282dcde
-  React: 1fe0eb13d90b625d94c3b117c274dcfd2e760e11
-  react-native-heap: 381fd78115a2a1314dae2906b893221c9501d12d
-  RNGestureHandler: afde2addc517e85cc97a25bfe119acbee779a7ce
-  yoga: b1ce48b6cf950b98deae82838f5173ea7cf89e85
+  React: 01aa04500b2957c2767e1dff9fe12e09444a467c
```

Comment:
Hashes changed - not too worried since we're really worried about tracking versions in npm rather than Cocoapods.  I wonder if there's a way to configure Cocoapods to do its thing without a lock file?  In this case, it just gets in our way, I think.